content/messages/messages-en.yaml

```diff
@@ -0,0 +1,116 @@
+---
+switch_to: "Switch to"
+hi_there: "Hi there!"
+choose_solutions: "Please choose between our solutions"
+the_pim: "The PIM"
+best_container: "The best container in town for your product information"
+onboarder: "The Onboarder"
+onboarder_description: "Easily manage product information from your suppliers"
+
+
+older_versions: "Older versions"
+can_we_help_you: "How can we help you?"
+type_your_search: "Type your search right here..."
+search_example: "For instance, try"
+what_family: "What is a family?"
+or: "or"
+projects: "Projects"
+how_permissions: "How do I set permissions?"
+using_akeneo: "Using<br>Akeneo"
+using_akeneo_description: "Discover Akeneo concepts. Master your products data. Be more productive. Work with your team."
+browse_all_articles: "Browse all articles"
+setting_up_the_pim: "Setting up<br>the PIM"
+setting_up_the_pim_description: "Master your catalog settings. Unleash the permissions power. Track your batch jobs."
+coding_around: "Coding<br>around"
+coding_around_description: "Enjoy the API magic. Deal with the customization kraken. Follow our styleguide."
+dev_hub_link: "Go to the dev hub"
+popular_guides: "Popular guides"
+need_more_help: "Need more help?"
+need_more_help_descrition: "Either you're having trouble setting up your PIM or modelizing your product data, The Akademy can give you a hand. Do not hesitate to contact us!"
+get_trained: "Get trained!"
+stay_in_touch_community: "Stay in touch with our open source community"
+slack_user_group: "Slack User Group"
+slack_user_group_description: "Come aboard! Join the Akeneo community"
+stack_overflow_description: "Need to find technical answers? Here you go!"
+crowdin_description: "Go multinational! Help us translate the PIM in other languages"
```

Comment:
I'd rather user "Go global" instead of multinational, multinational is used for corporations.